mRides-app/Mappers/UserMapper.cs

```diff
@@ -146,5 +146,21 @@ public void setGender(int id, string newGender)
         {
             userGateway.setGender(id, newGender);
         }
+
+        /**
+        * Set the theme of an activity depending on whether the user is a driver or a rider
+        */
```

Comment:
This is from a mapper class, I followed how it is for now in that class since I wasn't the one who wrote the rest of the code/comments in that class, but we are eventually going to change all the comments for older classes soon!